<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mycard</class>
 <widget class="QDialog" name="mycard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="cardlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>540</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BindBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定银行卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>解绑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>支付</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
